back-up-and-restore-data.md

```diff
@@ -8,8 +8,8 @@ The `cockroach dump` [command](cockroach-commands.html) outputs the SQL statemen
 
 When `cockroach dump` is executed:
 
-- Table schema and data are dumped as they appears at the time that the command is started. Any changes after the command starts will not be included in the dump.
-- If the dump takes longer than the [`ttlseconds`](configure-replication-zones.html) replication setting for the table (24 hours by default), the dump may fail. 
+- Table schema and data are dumped as they appear at the time that the command is started, or before the time specified in the `--as-of` flag. Any changes after the command starts, or at or after the timestamp specified in `--as-of`, will not be included in the dump.
```

Comment:
`before the time specified` -> `at the time specified`
`or at or after the timestamp` -> `or after the timestamp`